xml/gl.xml

```diff
@@ -43292,6 +43292,11 @@ typedef unsigned int GLhandleARB;
                 <command name="glDepthBoundsEXT"/>
             </require>
         </extension>
+        <extension name="GL_EXT_depth_clamp" supported="gles2">
+            <require>
+                <enum name="GL_DEPTH_CLAMP"/>
```

Comment:
I *think* this should be `GL_DEPTH_CLAMP_EXT`, no?

...Which would also require a line of this sort further up in this file (after the non-EXT one or so, I would think):
`<enum value="0x864F" name="GL_DEPTH_CLAMP_EXT"/>`

At least I don't think I've ever seen an EXT extension use a non-suffixed enum before... But then again, EXT extensions usually pop up before the functionality goes into the core, so perhaps that dance is pointless here...?

Reply:
I've changed it accordingly, thanks.